src/pfe/portal/routes/projects/links.route.js

```diff
@@ -270,4 +275,50 @@ function checkIfEnvsExistInArray(project, array) {
   return envPairs.every(env => array.includes(env));
 }
 
+
+async function restartAppsodyKubernetes(user, project) {
```

Comment:
Should we have extension specific code here? Does this belong in the appsody extension?

Reply:
Already added the code to the Appsody extension, unfortunately Appsody just needs to be updated differently in Kuberentes. Think the function should be `restartExtension` as each extension will need to be restarted using its extension framework rather than using Codewind specific stuff.